packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/queued_item_store/drift/drift_queued_item_store.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'dart:async';
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/queued_item_store.dart';
+import 'package:amplify_db_common_dart/amplify_db_common_dart.dart';
+import 'package:drift/drift.dart';
+
+part 'drift_queued_item_store.g.dart';
+
+/// SQL schema of data stored in DriftDatabase
+class DriftStrings extends Table {
+  /// Identifies object in the SQL database
+  IntColumn get id => integer().autoIncrement()();
+
+  /// The string value stored for this object
+  TextColumn get value => text()();
+}
+
+/// {@template amplify_analytics_pinpoint_dart.drift_queued_item_store}
+/// Drift class for managing stored [DriftStrings]
+/// {@endtemplate}
+@DriftDatabase(tables: [DriftStrings])
+class DriftQueuedItemStore extends _$DriftQueuedItemStore
+    implements QueuedItemStore {
+  /// {@macro amplify_analytics_pinpoint_dart.drift_queued_item_store}
+  factory DriftQueuedItemStore(String storagePath) {
+    final driftQueryExecutor = connect(
+      name: 'analytics_cached_events',
+      path: storagePath,
+    );
+    return DriftQueuedItemStore._(driftQueryExecutor);
+  }
+
+  DriftQueuedItemStore._(super.driftQueryExecutor);
+
+  // Reminder: Bump this number whenever you change or add a table definition.
+  @override
+  int get schemaVersion => 1;
+
+  /// Retrieve all [DriftString] stored in the database
+  Future<List<DriftString>> get allJsonStrings => select(driftStrings).get();
```

Comment:
I don't see this used. Am I missing it?

Similar to other comments, would rename this to `allStrings`. They happen to be json strings, this class is not serializing data to/from json. All this class knows is that it stores strings.

Reply:
Thanks it is legacy and can be safely deleted.